UTKcataloging/mods_to_rdf
# Add overall table, tables for elements without them,  fix predicates/URIs

**JIRA Issue**: [DIGITAL-925](https://jirautk.atlassian.net/browse/DIGITAL-925)

What Does this Do?
==================

This PR attempts to prepare all namespace / predicate tables for our final deliverable. A few additions have been made. Tables for the following top-level elements have been added as they weren't given in the initial mapping:

1. identifier
2. physicalDescription
3. typeOfResource
4. classification
5. recordInfo
6. accessCondition

The namespace section of the document has also been added to so that all namespaces present in the mapping document are represented.

Finally, a few smaller edits were made. For DBPedia, "dbpedia" rather than "dbo" had been used as the prefix in a few instances. In addition, some final forward slashes were added to a few of the URIs (like LoC identifiers URL - http://id.loc.gov/vocabulary/identifiers/ ). In addition, an attempt was made to standardized the predicate tables we include throughout.

How Should This Be Tested?
==========================

Answer the following:

1. Are all namespaces present in the final mapping (including relatedItem) represented in the Namespace section?
2. Do all of the top-level elements have a predicate table and are all the predicates represented?
3. Are there any instances in which a range should have been included but wasn't (currently we don't specify range for any property)?
4. Are any additional clarifications to the Usage Notes within the tables needed?
5. Is the correct value type for each predicate indicated?

Additional Notes
================

If we determine as a group that range is essentially "n/a" for all properties in our mapping, I would like to go ahead and remove that column from the tables. Please comment on this so I can take this action.
